proxy/http/HttpBodyFactory.cc

```diff
@@ -301,14 +299,13 @@ HttpBodyFactory::reconfigure()
 
   ats_scoped_str directory_of_template_sets(RecConfigReadConfigPath("proxy.config.body_factory.template_sets_dir", "body_factory"));
 
-  directory_of_template_sets = Layout::get()->relative(s);
+  directory_of_template_sets = Layout::get()->relative(directory_of_template_sets);
```

Comment:
At this point, we don't really need to do `relative_to` since `RecConfigReadConfigPath` guarantees us an absolute path.